doc/source/examples/incompressible-flow/2d-taylor-couette-flow/2d-taylor-couette-flow.rst

```diff
@@ -0,0 +1,280 @@
+    set output name             = couette
+    set subdivision             = 2
+    set number mesh adapt       = 2      # If steady, nb mesh adaptation
+  end
+
+
+Mesh adaptation
+----------------
+
+Mesh adaptation is quite complex in Lethe. The mesh can be dynamically adapted using Kelly error estimates on the velocity, pressure or variables arising from other physics. Lethe also supports uniform mesh refinement. Since we wish to measure the convergence of the error with respect to an analytical solution, we specify a uniform mesh refinement by setting ``type=uniform``
+
+.. code-block:: text
+
+  subsection mesh adaptation
+    set type                    = uniform
+  end
+
+
+Forces
+-------
+
+The ``forces`` subsection controls the postprocessing of the torque and the forces acting on the boundaries of the domain.
+
+.. code-block:: text
+
+  subsection forces
+      set verbosity             = verbose   # Output force and torques in log <quiet|verbose>
+      set calculate torques     = true     # Enable torque calculation
+  end
+
+By setting ``calculate torques=true``, the calculation of the torque resulting from the fluid dynamics physics on every boundary of the domain is automatically calculated. Setting ``verbose=verbose`` will print out the value of the torque calculated for each mesh. 
+
+
+Rest of the subsections
+------------------------
+
+The non-linear and linear solvers subsection do not contain any new information in this example.
+
+Running the simulation
+----------------------
+Launching the simulation is as simple as specifying the executable name and the parameter file. Assuming that the ``gls_navier_stokes_2d`` executable is within your path, the simulation can be launched by typing:
+
+.. code-block:: text
+
+  gls_navier_stokes_2d taylor-couette.prm
+
+Lethe will generate a number of files. The most important one bears the extension ``.pvd``. It can be read by popular visualization programs such as `Paraview <https://www.paraview.org/>`_. 
+
+Results
+---------------------------
+
+Using Paraview, the steady-state velocity profile can be visualized:
+
+.. image:: images/flow_patterns.png
+    :alt: velocity distribution
+    :align: center
+
+As can be seen, each cell is curved because a Q2 isoparametric mapping was used (by setting ``qmapping=true`` in the FEM subsection). To visualize these high-order cells, we need to subdivide the regular cell to store additional information onto them. A good practice is to use as many subdivision as the interpolation order of the scheme. Hence, we used ``subdivision=2`` in the simulation control subsection. Finally, by default, paraview does not render high-order elements. To enable the rendering of high-order elements, the Nonlinear subdivision level slider must be increased above one. For more information this topic, please consult the deal.II wiki page on `rendering high-order elements <https://github.com/dealii/dealii/wiki/Notes-on-visualizing-high-order-output>`.
```

Comment:
```suggestion
As can be seen, each cell is curved because a Q2 isoparametric mapping was used (by setting ``qmapping=true`` in the FEM subsection). To visualize these high-order cells, we need to subdivide the regular cell to store additional information onto them. A good practice is to use as many subdivisions as the interpolation order of the scheme. Hence, we used ``subdivision=2`` in the simulation control subsection. Finally, by default, Paraview does not render high-order elements. To enable the rendering of high-order elements, the Nonlinear subdivision level slider must be increased above one. For more information on this topic, please consult the deal.II wiki page on `rendering high-order elements <https://github.com/dealii/dealii/wiki/Notes-on-visualizing-high-order-output>`.
```